docs/docs/configuration/kafka-and-zookeeper.md

```diff
@@ -0,0 +1,90 @@
+kafka.broker.list              | list of all brokers in the cluster (or at least some contact points); separated with ',' | localhost:9092
+kafka.namespace                | namespace is a prefix prepended to all Kafka topics and consumer groups used by Hermes   | <empty>
+kafka.zookeeper.connect.string | [Consumers only] connection string to Kafka Zookeeper                                    | localhost:2181
+kafka.cluster.name             | name of Kafka cluster (relevant only when connecting to multiple clusters)               | primary
+
+Zookeeper connection specific options (retries etc) are read from Metadata Zookeeper options.
+
+Management module can connect to multiple Kafka clusters at once (see [section below](#multiple-kafka-clusters)), thus
+when specifying connection option is done per cluster. Simple configuration for single cluster looks following:
+
+```yaml
+kafka:
+  defaultNamespace: // namespace shared by all clusters, default: <empty>
+  clusters:
+    -
+      clusterName: // name of cluster, can be any arbitrary string, default: primary
+      connectionString: // connection string to cluster Zookeeper, default: localhost:2181
+```
+
+### Multiple Kafka clusters
+
+Hermes can be configured to publish and read messages to/from multiple Kafka clusters. This is the scenario we use for
+having our Kafka clusters in different data centers. If Kafka in one DC fails, whole traffic can be routed to the second
+DC. This scenario assumes, that Kafka clusters hold different set of messages. There is no support for multiple clusters
+each holding the same copy of data.
+
+This is the schematics of two data center architecture:
+
+![Multi DC schematics](/img/architecture-multi-cluster.png)
+
+* there are specific **Frontend** and **Consumers** instances per cluster:
+    * each **Frontend** instance writes data to single cluster
+    * each **Consumers** instance reads data from single cluster
+* **Management** connects to all Kafka clusters and keeps topics and subscriptions in-sync
+* all **Frontend**, **Consumers** and **Management** instances connect to the same Metadata Zookeeper, regardless of cluster they
+    connect to
+
+Configuring Frontend and Consumers is easy: use configuration options from [previous chapter](#single-kafka-cluster) to
+connect to given clusters. Remember about specifying proper `kafka.cluster.name`.
+
+Since Management instances need to know all clusters, their configuration is bit more complex. Example configuration for
+the schematics provided above:
+
+```yaml
+kafka:
+  clusters:
+    -
+      clusterName: dc1
+      connectionString: kafka-zookeeper:2181/clusters/dc1
+    -
+      clusterName: dc2
+      connectionString: kafka-zookeeper:2181/clusters/dc2
+```
+
+### Multiple Hermes on single Kafka cluster
+
+It is also possible to run multiple Hermeses on single Kafka cluster, e.g. to separate different test environments. The
```

Comment:
i think instance is more like Frontend, Consumers etc - here i mean multiple hermes clusters